contact[1].php

```diff
@@ -10,21 +20,40 @@
 		<meta name="author" content="Johno Devine">
 		<meta name="viewport" content="width=device-width, initial-scale=1">
 		<link rel="stylesheet" href="stylesheet.css"/>
+		<link rel="icon" type="image/x-icon" href="images/icon.ico">
 	</head>
 	<body>
+		<!-- background-->
 		<div class="leftbackground"></div>
 		<div class="rightbackground"></div>
+		<!-- checks if the user logged in is an admin or not-->
+		<?php
+		if(!isset($_SESSION['Form'])){
+			 ?>
+			<button class="btn" onclick="window.location.href='login.php';"> <img src="images/profile.png" class="profile_img"/></button>
+			<?php
+		}
+		elseif ($_SESSION['Form'] != 'Graeme'){
+			?>
+			<button class="logout_btn" onclick="window.location.href='logout.php';"> Log Out </button>
+			<?php
+		}
+
+		else {
+			?>
 			<button class="btn" onclick="window.location.href='profile.php';"> <img src="images/profile.png" class="profile_img"/></button>
-		<div id="myNav" class="overlay">
-		  <a href="javascript:void(0)" class="closebtn" onclick="closeNav()"><span style="color: #ff244f;">&times;</span></a>
-		  <div class="overlay-content">
```

Comment:
I change my nav into its own php file since it was a common code that was inside all of my pages